cs-engine/install.md

```diff
@@ -101,33 +101,18 @@ to the `docker` group.
     $ sudo apt-get update && sudo apt-get install apt-transport-https
     ```
 
-4.  Install additional virtual drivers not in the base image.
+4.  Install additional kernel modules to add AUFS support.
 
     ```bash
-    $ sudo apt-get install -y linux-image-extra-virtual
+    $ sudo apt-get install -y linux-image-extra-$(uname -r) linux-image-extra-virtual
     ```
 
-    You may need to reboot your server after updating the LTS kernel.
-
 5.  Add the repository for the new version:
 
     ```bash
     $ echo "deb https://packages.docker.com/1.12/apt/repo ubuntu-trusty main" | sudo tee /etc/apt/sources.list.d/docker.list
     ```
 
-    This adds the repository of the latest version of CS Docker Engine for the
-    Ubuntu Trusty distribution. Change the "ubuntu-trusty" string to the
-    distribution you're using:
-
-    * debian-jessie (Debian 8)
```

Comment:
According to http://success.docker.com/Get_Help/Compatibility_Matrix, debian is not supported. @cpuguy83 or @mavenugo can you confirm this?